update code transforms the support new api

shivdeep-singh-ibm opened the pull request:
## Why are these changes needed?


## Related issue number (if any).

shivdeep-singh-ibm committed:
update code transforms ti support new api

Signed-off-by: Shivdeep Singh <Shivdeep.Singh@ibm.com>

shivdeep-singh-ibm committed:
Add simpler api notebooks

Signed-off-by: Shivdeep Singh <Shivdeep.Singh@ibm.com>

shivdeep-singh-ibm committed:
fix ci/cd for new api code patch

Signed-off-by: Shivdeep Singh <Shivdeep.Singh@ibm.com>

touma-I reviewed (approved):
@shivdeep-singh-ibm thanks you being thorough with this migration.

revit13 commented on transforms/code/code2parquet/Dockerfile.ray:
```diff
@@ -1,41 +1,27 @@
 # see https://docs.openshift.com/container-platform/4.17/openshift_images/create-images.html#use-uid_create-images
 USER root
-RUN chown ray:root /home/ray && chmod g=u /home/ray
+RUN chown ray:root /home/ray && chmod 775 /home/ray
```
We encountered permissions issues running on OpenShift and using `g=u` fixed that according to this [link](https://docs.openshift.com/container-platform/4.17/openshift_images/create-images.html#use-uid_create-images). I suggest testing it on Openshift cluster first... Thanks  cc @roytman

shahrokhDaijavad commented:
@touma-I Notebooks are all there, but they are the absolute minimum! README's are missing. I will work on the READMEs.

shivdeep-singh-ibm committed:
Update README.md and notebooks for code transforms

- update code2parquet, license_select and proglang-select with README and notebooks.

Signed-off-by: Shivdeep Singh <Shivdeep.Singh@ibm.com>

touma-I committed:
merge with latest releave from dev

Signed-off-by: Maroun Touma <touma@us.ibm.com>

shahrokhDaijavad committed:
Code2parquet README and all IBM to data-prep-kit org stuff

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Change data-prep-lab to data-prep-kit

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

shahrokhDaijavad committed:
Merge branch 'dev' into new-api-code-transforms

touma-I reviewed (approved):
We should merge this PR and do any additional iterations in separate PRs

touma-I committed:
DCO problem

Signed-off-by: SHAHROKH DAIJAVAD <shahrokh@us.ibm.com>

touma-I committed:
Merge branch 'data-prep-kit:dev' into new-api-code-transforms

shahrokhDaijavad commented:
Hi, @shivdeep-singh-ibm. With the new ownership of DPK by the Linux Foundation, we had to change some of the rules for the repo, so now it doesn't take "unverified" commits before a PR can be merged. I had made some changes to your PR, when my signature was not verified. I have a verified signature now, but because of my old unverified signatures in this PR, it cannot be merged. Can you please create a "new" PR, copy everything that is in this PR, and submit it again with a verified signature? Thanks. Maroun is ready to merge it.

touma-I commented:
Replace with another PR that has verified signature
